webrtc.html

```diff
@@ -4414,37 +4414,43 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <dt>checking</dt>
 
           <dd>The <code><a>RTCIceTransport</a></code> has received at least one
-          remote candidate, and is checking candidate pairs but has not yet
-          found a connection. In addition to checking, it may also still be
-          gathering.</dd>
+          remote candidate, and either is checking candidate pairs but has not yet
```

Comment:
It looks like I suggested putting the "either" in the wrong spot; my apologies. The intended logic is "received candidate AND is checking AND (no connection found OR consent checks failed)"